src/data/calculate.ts

```diff
@@ -360,25 +362,39 @@ const getVaccineMultiplier = (data: CalculatorData): number | null => {
   return vaccineMultiplierPerDose[data.yourVaccineDoses]
 }
 
-export const calculateActivityRisk = (data: CalculatorData): number | null => {
+export const getActivityRiskVariables = (
+  data: CalculatorData,
+): Array<{ name: string; value: number }> => {
   try {
     if (data.interaction === '') {
-      return null
+      return []
     }
-
     const vaccineMultiplier = getVaccineMultiplier(data)
     if (vaccineMultiplier === null) {
-      return null
+      return []
     }
 
+    const forWord = i18n.t('for')
```

Comment:
Using single words like this is a little hacky, happy to do a more thorough translation job later but am out of energy and wanted to get this up for initial review